### PR TITLE
Simple tests for origin-when-cross-origin referrer policy

### PR DESCRIPTION
**Pull request by youennf**

### CODE REVIEW

File: fetch/api/basic/referrer.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  importScripts("../resources/get-host-info.sub.js")
+}
+
+function runTest(url, init, expectedReferrer, title) {
+    promise_test(function(test) {
+        return fetch(url , init).then(function(resp) {
+            assert_equals(resp.status, 200, "HTTP status is 200");
+            assert_equals(resp.headers.get("x-request-referer"), expectedReferrer, "Request's referrer is correct");
+        });
+  }, title);
```
Comment (jdm) [inline fetch/api/basic/referrer.js]: nit: indentation.

File: fetch/api/basic/referrer.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+var fetchedUrl = RESOURCES_DIR + "inspect-headers.py?headers=referer";
+var corsFetchedUrl = get_host_info().HTTP_REMOTE_ORIGIN  + dirname(location.pathname) + RESOURCES_DIR + "inspect-headers.py?headers=referer&cors";
+var redirectUrl = RESOURCES_DIR + "redirect.py?location=" ;
+var corsRedirectUrl = get_host_info().HTTP_REMOTE_ORIGIN + dirname(location.pathname) + RESOURCES_DIR + "redirect.py?location=";
+
+runTest(fetchedUrl, { referrerPolicy: "origin-when-cross-origin"}, location.toString(), "origin-when-cross-origin policy on a same-origin URL");
+runTest(corsFetchedUrl, { referrerPolicy: "origin-when-cross-origin"}, get_host_info().HTTP_ORIGIN + "/", "origin-when-cross-origin policy on a cross-origin URL");
```
Comment (jdm) [inline fetch/api/basic/referrer.js]: nit: either remove the space after { or add one before }.

File: fetch/api/basic/referrer.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            assert_equals(resp.status, 200, "HTTP status is 200");
+            assert_equals(resp.headers.get("x-request-referer"), expectedReferrer, "Request's referrer is correct");
+        });
+  }, title);
+}
+
+var fetchedUrl = RESOURCES_DIR + "inspect-headers.py?headers=referer";
+var corsFetchedUrl = get_host_info().HTTP_REMOTE_ORIGIN  + dirname(location.pathname) + RESOURCES_DIR + "inspect-headers.py?headers=referer&cors";
+var redirectUrl = RESOURCES_DIR + "redirect.py?location=" ;
+var corsRedirectUrl = get_host_info().HTTP_REMOTE_ORIGIN + dirname(location.pathname) + RESOURCES_DIR + "redirect.py?location=";
```
Comment (jdm) [inline fetch/api/basic/referrer.js]: redirectUrl and corsRedirectUrl aren't used.

**Comment by youennf** [comment]
@jdm, thanks.
I updated the file according your comments

**Comment by jdm** [comment]
Thanks!

**Pull request merged**